package.json

```diff
@@ -20,15 +20,21 @@
     "url": "git://github.com/edwellbrook/node-tvdb.git"
   },
   "dependencies": {
-    "jszip": "3.1.2",
-    "request": "2.74.0",
-    "xml2js": "0.4.17"
+    "debug": "^2.2.0",
+    "lodash": "^4.15.0",
```

Comment:
Conscious of adding many dependencies here. Don't mind lodash being added as a dev dep for tests, however, if we're only using it for two functions in the actual library can we either do things ourselves, or require the exact packages (`require('lodash/defaults')`).

for reference: http://knpw.rs/blog/using-lodash